gradle-xmake-plugin/src/main/resources/lua/install_artifacts.lua

```diff
@@ -41,7 +41,7 @@ end
 function _install_artifacts(targets, opt)
     local arch = opt.arch
     local installdir = opt.installdir
-    assert(xmake.version():satisfies(">= 2.9.5"), "please update xmake to >= 2.9.5")
+    assert(xmake.version():satisfies("> 2.9.5"), "please update xmake to > 2.9.5")
```

Comment:
xmake v2.9.6 has not been released yet.

Reply:
Yes, I know. 
At first I thought the patch for the install function was included in v2.9.5, so that's why I was testing for equivalence, but as it wasn't released for that version I had to remove equivalence.
Is it a problem to test on a version that has not been released? If so, this PR can wait until 2.9.6 is released.

Reply:
you can wait for 2.9.6